theforeman.org/yaml/jobs/test_plugin_pipeline.yaml

```diff
@@ -0,0 +1,135 @@
+# parameters:
+#   name: plugin name, e.g. foreman_bootdisk
+#   repo: git repo name as it may differ, e.g. foreman_bootdisk
+#   branch: git repo branch to test, e.g. master
+#   foreman_branch: foreman git repo branch to run on, e.g. develop
+- job-template:
+    name: 'test_foreman_plugin_{plugin}_{branch}'
```

Comment:
Would be nice here to update to newer naming conventions as part of the shift. I think it would translate to:

```
foreman-plugin-{plugin}-{branch}-test
```

foreman-plugin-discovery-develop-test
foreman-plugin-ansible-master-test
foreman-plugin-remote_execution-master-test